tests/core/pyspec/eth2spec/test/merge/genesis/test_initialization.py

```diff
@@ -120,3 +120,55 @@ def test_initialize_post_transition(spec):
     assert spec.is_merge_complete(state)
 
     yield 'state', state
+
+
+@with_merge_and_later
```

Comment:
we'll also want to add a unittest to `phase0/genesis`

Reply:
Added a test in 70e179f7